manual.typ

```diff
@@ -1130,147 +748,177 @@ The palette library provides some predefined palettes.
 
-==== Styling
+    // Resolve the current style ("star")
+    let style = cetz.styles.resolve(ctx.style, style.named(),
+      base: def-style, root: "star")
 
-#def-arg("amplitude", `<number>`, default: decorations.flat-brace-default-style.amplitude, [Determines how much the brace rises above the base line.])
-#def-arg("aspect", `<number>`, default: decorations.flat-brace-default-style.aspect, [Determines the fraction of the total length where the spike will be placed.])
-#block(breakable: false, def-arg("curves", `<array> or <number>`, default: decorations.flat-brace-default-style.curves, [
-  Customizes the control points of the curved parts.
-  Setting a single number is the same as setting ```typc (num, auto, auto, auto)```.
-  Setting any item to #auto will use its default value.
-  The first item specifies the curve widths as a fraction of the amplitude.
-  The second item specifies the length of the green and blue debug lines as a fraction of the curve's width.
-  The third item specifies the vertical offset of the red and purple debug lines as a fraction of the curve's height.
-  The fourth item specifies the horizontal offset of the red and purple debug lines as a fraction of the curve's width.
-]))
-#def-arg("outer-curves", `<array> or <number> or <auto>`, default: decorations.flat-brace-default-style.outer-curves, [
-  Customizes the control points of just the outer two curves (just the blue and purple debug lines).
-  Overrides settings from `curves`.
-  Setting the entire value or individual items to #auto uses the values from `curves` as fallbacks.
-])
-#def-arg("content-offset", `<number>`, default: decorations.flat-brace-default-style.content-offset, [Offset of the `content` anchor from the spike.])
-#def-arg("debug-text-size", `<length>`, default: decorations.flat-brace-default-style.debug-text-size, [Font size of displayed debug points when `debug` is #true.])
+    // Compute the corner coordinates
+    let corners = range(0, style.n * 2).map(i => {
+      let a = 90deg + i * 360deg / (style.n * 2)
+      let r = if calc.rem(i, 2) == 0 { style.radius } else { style.inner-radius }
 
-==== Default `flat-brace` Style
-#decorations.flat-brace-default-style
+      // Output a center relative coordinate
+      (rel: (calc.cos(a) * r, calc.sin(a) * r, 0), to: center)
+    })
 
-= Advanced Functions
+    line(..corners, ..style, close: true)
+  })
+}
 
-== Coordinate
+// Call the element
+my-star((0,0))
+my-star((0,3), n: 10)
 
-#let coord-module = tidy.parse-module(read("src/coordinate.typ"), name: "Coordinate")
-#show-module-fn(coord-module, "resolve")
+set-style(star: (fill: yellow)) // set-style works, too!
+my-star((0,6), inner-radius: .3)
+```
+
+= Internals
+== Context
+
+The state of the canvas is encoded in its context object. Elements or other
+draw calls may return a modified context element to the canvas to change its
+state, e.g. modifying the transformating matrix, adding a group or setting a style.
```

Comment:
Call it just a "dictionary" instead of object or element. Also mention where it can be found in `/src/canvas.typ` I don't think we should worry about documenting its keys atm.